package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www",
+    "test": "jest",
+    "e2e-test": "jest --config jest.e2e.config.js",
+    "lint": "eslint .",
+    "fix": "eslint --fix ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/software-developer-org/website.git"
+  },
+  "author": "Tai Truong Consulting",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/software-developer-org/website/issues"
+  },
+  "dependencies": {
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "eslint": "^7.19.0",
```

Comment:
Some packages should be defined as devDependencies

Reply:
Splitted in dependencies and devDependencies.
 
"dependencies" are set by default through express-generator 
-   "cookie-parser"
 -   "debug"
 -   "express"
 -   "http-errors"
 -   "morgan"
 -   "pug"
  
"devDependencies":
 -   "eslint"
 -   "eslint-config-prettier"
 -   "eslint-plugin-jest-playwright"
 -   "jest"
 -   "jest-playwright-preset"
 -   "nodemon"
 -   "playwright"
 -   "prettier"